src/protocols/WebSocket.ts

```diff
@@ -292,14 +273,16 @@ export default class WebSocketProtocol extends BaseProtocolRealtime {
             this.emit('websocketRenewalDone'); // Notify that the websocket has finished renewing his connection with Kuzzle
           });
       })
-      .catch(error => this.emit(formattedRequest.payload.requestId, {error}));
-
+      .catch(error => {
```

Comment:
nit: should we asyncify this function call?

Reply:
I dont see why